CHANGELOG.md

```diff
@@ -1,3 +1,62 @@
+# 1.6.0-rc01 (February 2024)
```

Comment:
Let's explicitly mention that this list of changes is compared to beta02

Reply:
Done